tests/ui/wildcard_imports_2021.stderr

```diff
@@ -1,132 +1,73 @@
-error: usage of wildcard import
-  --> $DIR/wildcard_imports_2021.rs:9:5
+error[E0463]: can't find crate for `wildcard_imports_helper`
```

Comment:
This one was real confusing, it looks like it's the 2021 revision of the test but it's not, the test file has 2021 in its name **and** two edition revisions

The `.stderr` must have been left behind from a previous version that didn't have revisions making it trickier by showing a diff instead of a new file creation